SECURITY_UPDATES_TODO.md

```diff
@@ -0,0 +1,392 @@
+# Security Updates TODO
+
+**Date Created**: 2025-10-29
+**Status**: Pending - To be addressed after PR #605 is merged
+**Total Vulnerabilities**: 43 (3 critical, 10 high, 24 moderate, 6 low)
```

Comment:
This is waiting on python version upgrade, which is waiting for Otava to do that first.